Sources/TuistCache/Utilities/XcodeProjectPathHasher.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import CryptoKit
+
+// Thanks to https://pewpewthespells.com/blog/xcode_deriveddata_hashes.html for
+// the initial Objective-C implementation.
+// This is taken from XCLogParser, from Spotify, at:
```

Comment:
What should we do with the license since `XCLogParser` is Apache and not MIT? Have we already encountered this situation @pepibumur ?

If you want to be sure, I can ask to Spotify folks about it (they wrote XCLogParser)